src/formatDate.js

```diff
@@ -50,7 +50,38 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[3]);
+  const result = [];
+
+  for (let i = 0; i < 3; i++) {
+    const item = toFormat[i];
+    const index = fromFormat.indexOf(item);
+
+    if (index >= 0) {
+      result.push(dateArray[index]);
+    }
+
+    if (index === -1 && item === 'YYYY') {
+      const indexOfYear = fromFormat.indexOf('YY');
+
+      if (dateArray[indexOfYear] < 30) {
+        result.push('20' + dateArray[indexOfYear]);
+      } else {
+        result.push('19' + dateArray[indexOfYear]);
```

Comment:
The difference is only 19 or 20. You can easily use ternary operator here to improve your code a bit